packages/farcaster-js-hub-rest/package.json

```diff
@@ -50,7 +50,8 @@
     "clean:dist": "rm -rf dist",
     "clean:generated": "rm -rf src/openapi/generated",
     "cz": "yarn run -T cz",
-    "fixup-imports": "yarn run -T fix-esm-import-path src && sed -i.bak 's|models\"|models/index.js\"|' src/openapi/generated/**/*.ts && sed -i.bak 's|models|models/index.js|' src/openapi/generated/apis/**/*.ts && rm src/openapi/generated/**/*.ts.bak",
+    "fixup-imports": "yarn run -T fix-esm-import-path src && yarn run fix-utils && sed -i.bak 's|models\"|models/index.js\"|' src/openapi/generated/**/*.ts && sed -i.bak 's|models|models/index.js|' src/openapi/generated/apis/**/*.ts && rm src/openapi/generated/**/*.ts.bak",
+    "fix-utils": "sed -i.bak 's/@noble\\/hashes\\/utils\\.js/@noble\\/hashes\\/utils/g' src/utils.ts && rm src/utils.ts.bak",
```

Comment:
This is curious. `src/utils.ts` was a hand-written file, so why does it need a script to edit one line?

Also, why does that line need to be edited as such?